packages/roosterjs-react/lib/contextMenu/types/ContextMenuItemStringKeys.ts

```diff
@@ -70,6 +70,15 @@ export type TableEditDeleteMenuItemStringKey =
     | 'menuNameTableDeleteColumn'
     | 'menuNameTableDeleteRow';
 
+/**
+ * Key of localized strings of Table Options menu items
+ */
+export type TableOptionsMenuItemStringKey =
```

Comment:
Suggest to add this type in demo site only, but no need to update roosterjs-react for now.

Reply:
Ok, but is there a reason those types are a part of roosterjs-react when they are only used on the demo site?

Reply:
For the other types, they are used by table context menu in roosterjs-react.